environment.yml

```diff
@@ -4,6 +4,7 @@ channels:
 dependencies:
   - python=3.8 # nail the python version, so conda does not try upgrading / downgrading
   - conda-forge::ctapipe=0.12
+  - numpy=1.22.4 # higher versions don't work due to astropy4 dependency in ctapipe0.12
```

Comment:
I'd suggest not pinning to `1.22.4` as that would prevent updating to a possible 1.22.5 bug-fix release

Reply:
Yes, thanks.  Trying to modify the commit now...